widgetssdk/src/main/java/com/glia/widgets/call/CallView.java

```diff
@@ -444,6 +431,40 @@ public void startVisitorVideoView(MediaState visitorMediaState) {
                 .getScreenSharingController(screenSharingCallback);
     }
 
+    private void shouldShowVisitorVideo(CallState state) {
+        if (
+                state.is2WayVideoCall() &&
+                        state.callStatus
+                                .getVisitorMediaState()
+                                .getVideo()
+                                .getStatus() == Media.Status.PLAYING &&
+                        visitorVideoContainer.getVisibility() == GONE
+        ) {
+            visitorVideoContainer.setVisibility(VISIBLE);
+            showVisitorVideo(state.callStatus.getVisitorMediaState());
+        }
+        if (!state.is2WayVideoCall() && visitorVideoContainer.getVisibility() == VISIBLE) {
```

Comment:
This if could be an else if. If I understand correctly, either only the first or only the second can happen. Same in `shouldShowOperatorVideo`

Reply:
renamed to handleOperatorVideoState